### ISSUE TITLE
Setting subject for testnewsletters does not work

### ISSUE THREAD

**Issue by responseinformationsdesign** [kartolo/direct_mail#28]
Setting value for mod.web_modules.dmail.testmail has no effect.

**Comment by kartolo** [comment]
do you set the "send as test mail" checkbox in the last step?

**Comment by responseinformationsdesign** [comment]
This was my misunderstanding. I exspected that this option would influence the test messages sent in step 4. Wouldn´t this checkbox better be placed there?

**Comment by kartolo** [comment]
no, it was a request, for editor who needs to send test mail from the last step. Some user doesn't understand how to set the direct_mail parameter for the step 4 and they wrote me, that the JumpURL of test mail doesn't work, because they send the mail using the simple test.

Maybe it's better to drop the whole step 4?

**Comment by responseinformationsdesign** [comment]
I don´t think so. For our customer it´s very important to have the two steps sending testmail for editors and sending mails to subscribers divided in seperate steps. It would be too dangerous that editors would mix up the distribution groups in the last step.

What about integrating this checkbox in step 4 also or always change the mail subject for messages sent in step 4?

**Comment by kartolo** [comment]
> What about integrating this checkbox in step 4 also or always change the mail subject for messages sent in step 4?

I guess it would be better to always set the prefix on step 4. opinion?

**Comment by responseinformationsdesign** [comment]
I agree as this would be the assumed behaviour when setting a value for mod.web_modules.dmail.testmail.
